services/tables/src/main/java/com/linkedin/openhouse/tables/api/spec/v0/request/components/Policies.java

```diff
@@ -48,4 +48,11 @@ public class Policies {
       example = "{replication:{config:[{destination: clusterA, interval: 12H}]}}")
   @Valid
   Replication replication;
+
+  @Schema(
+      description =
+          "History as required in /tables API request. This field holds the snapshot retention specification.",
```

Comment:
Is this accurate though ? If you don't provide `History` it is totally fine right

Reply:
I'll modify this comment good catch